### ISSUE TITLE
Posting attachments to Autotask

### ISSUE THREAD

**Issue by natchu** [MattParr/python-atws#65]
Hi,
I am trying to post attachments to Autotask. 
I am currently testing like below:
Assume `at` is my connection object, and successfully connected to my Autotask instance and API Version is 1.6
I have created an Attachment object 'attachment' locally as follows
``` 
  (Attachment)
  {
	Data = b 'data in binary', 
	Info =
		(AttachmentInfo) {
			Fields =
				(ArrayOfField) {
					Field[] = < empty >
				}
			id = None
			UserDefinedFields =
				(ArrayOfUserDefinedField) {
					UserDefinedField[] = < empty >
				}
			ParentID = 379293
			ParentType = 4
			Type = "FILE_ATTACHMENT"
			Title = "New-image.jpg"
			FullPath = None
			AttachDate = None
			FileSize = None
			AttachedByResourceID = None
			AttachedByContactID = None
			Publish = 1
			ContentType = "image/jpeg"
			OpportunityID = None
		}
   }
```
Then I am trying to call `at.CreateAttachment(attachment)`. This does not throw any error, but I can't see the attachment in the system as well. Any help as to how to post attachments would be really appreciated.

Thanks,
Natchu

**Comment by natchu** [comment]
I have managed to go further into this, and now get the error as below:
`E                        suds.WebFault: Server raised fault: 'System.Web.Services.Protocols.SoapException: Conversion from type 'XmlNode()' to type 'String' is not valid.
E                          at autotask.web.services.API.v1_6.ATWS.CreateAttachment(Attachment attachment)
E                          at AutotaskApiApplication.API.V1_6.AutotaskApiService_1_6.CreateAttachment(Attachment attachment)'
`
Being new to suds, it looks a bit tricky to me. I have tried and worked across by making the SOAP envelope look exactly like the one in Autotask docs, but still this fails. Should I set something in the client? Any help is appreciated.
